source/_integrations/zwave_js.markdown

```diff
@@ -130,17 +130,25 @@ While your Z-Wave mesh is permanently stored on your dongle, the additional meta
 2. Select the Z-Wave integration. Then select **Configure**.
 3. Select **Add device**.
    - The Z-Wave controller is now in inclusion mode.
-4. If your device supports SmartStart (700 series controller), select **Scan QR code** and scan the QR code on your device.
-5. If your device does not support SmartStart, set the device in inclusion mode. Refer to the device manual to see how this is done.
-   - If your device is included using S2 security, you may be prompted to enter a PIN number provided with your device. Often, this PIN is provided with the documentation _and_ is also printed on the device itself. For more information on secure inclusion, refer to [this section](/integrations/zwave_js/#should-i-use-secure-inclusion).
+4. Check, if your device supports SmartStart:
+   - On the packaging, check for the SmartStart label.
+   - Find the QR code. It can be on the packaging or on the device itself.
+5. Depending on whether your device supports SmartStart, follow the steps in either option 1 or 2:
+   - **Option 1: your device supports SmartStart**:
+     - Select **Scan QR code** and scan the QR code on your device.
+     - If scanning does not work (for example due to missing HTTPS), paste the QR code content as text from a different QR reader and select **Submit**.
+     - If you haven't done so yet, turn the device on.
```

Comment:
The device should ideally **not** be turned on until after scanning the QR code. If it is already on, it may require a power cycle to join.